tests/dummy/app/router.js

```diff
@@ -1,7 +1,9 @@
 import Ember from 'ember'
+const {Router: EmberRouter} = Ember
+
 import config from './config/environment'
 
-const Router = Ember.Router.extend({
+const Router = EmberRouter.extend({
```

Comment:
This was my own manual doing to overcome the variable named `Router` and destructuring `Router`. I prefer this to renaming the variable `MyRouter` and figured since I'm the author and primary maintainer of this project it should be okay. Let me know if you disagree.